typing.html

```diff
@@ -1,7 +1,9 @@
 <!DOCTYPE html>
 <html>
 <head>
+	<title>Typing Test</title>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<h1 align="center">Pangram</h1>

```

Comment:
questtext can be changed, so I think this h1 element is not needed. Thank you for your PR.